cedar-drt/fuzz/fuzz_targets/partial-eval.rs

```diff
@@ -15,31 +15,20 @@
  */
 
 #![no_main]
+use cedar_drt::utils::expr_to_est;
 use cedar_drt::*;
 use cedar_drt_inner::*;
-use cedar_policy_core::ast;
-use cedar_policy_core::ast::Policy;
-use cedar_policy_core::ast::PolicyID;
-use cedar_policy_core::ast::PolicySet;
-use cedar_policy_core::ast::Value;
-use cedar_policy_core::authorizer::AuthorizationError;
-use cedar_policy_core::authorizer::{Authorizer, PartialResponse, Response};
+use cedar_policy_core::ast::Expr;
 use cedar_policy_core::entities::Entities;
+use cedar_policy_generators::schema::arbitrary_schematype_with_bounded_depth;
 use cedar_policy_generators::{
-    abac::{ABACPolicy, ABACRequest},
-    err::Error,
-    hierarchy::HierarchyGenerator,
-    schema::Schema,
+    abac::ABACRequest, err::Error, hierarchy::HierarchyGenerator, schema::Schema,
     settings::ABACSettings,
 };
 use libfuzzer_sys::arbitrary::{self, Arbitrary, Unstructured};
 use log::debug;
+use log::info;
 use serde::Serialize;
-use smol_str::SmolStr;
-use std::collections::HashMap;
-use std::collections::HashSet;
-use std::convert::TryFrom;
-use std::sync::Arc;
 
 /// Input expected by this fuzz target:
 /// An ABAC hierarchy, policy, and 8 associated requests
```

Comment:
maybe comment needs updating. Not sure if it's valuable to list the fields in this comment, since you can just look at the public fields & their docstrings